lang/en/docs/tutorials/platform-access.json

```diff
@@ -0,0 +1,107 @@
+{
+    "descriptionLinks": [
+        "Various ways of accessing Mat3ra platform: https://docs.mat3ra.com/tutorials/overview/"
+    ],
+    "description": "In this short video, we will go through various ways of accessing Mat3ra platform. There are three main modes of accessing Mat3ra platform: (1) Web/browser interface, (2) Command line interface (CLI), and (3) API access.",
+    "tags": [
+        {
+            "...": "../../metadata/general.json#/tags"
+        },
+        "API",
+        "CLI"
+    ],
+    "title": "Mat3ra Tutorial: Various ways of accessing Mat3ra platform",
+    "youTubeCaptions": [
+        {
+            "text": "Hi, <break time='0.25'/> welcome to matera tutorial series once again. In this short video, we will give a glimpse of various ways of accessing our platform.",
+            "startTime": "00:00:00.000",
+            "endTime": "00:00:09.000"
+        },
+        {
+            "text": "There are three main ways to access our simulation platform, namely, <break time='0.25'/> the web interface using a browser, <break time='0.5'/> command line interface via S S H, <break time='0.25'/> and A P I access. These different modes of access are designed keeping in mind various use cases. For example, <break time='0.25'/> a novice user will probably find the web browser interface most user friendly to run his or her D F T simulations, <break time='0.25'/> while an enterprise user might find our A P I access is suitable to integrate our services with their existing client application. However, these different modes of access are complimentary to each other, and oftentimes, a user might want to use more than one mode or all three modes to interact with our platform depending on the task at hand.",
```

Comment:
![Screenshot 2024-10-10 at 3 12 29 PM](https://github.com/user-attachments/assets/2a18c8d3-91bc-477a-a249-6d76a82d60ee)
When we use long text snippets like this, it's hard to follow the video with captions. Could we split?

Reply:
For the current video, I have regenerated the subtitles using YouTube's automatic subtitle feature. In the future, we should have smaller text segments.